admin/views/pupil-status/view-pupil-status.ejs

```diff
@@ -354,7 +351,7 @@
                               <button class="app-step-nav__button app-step-nav__button--title js-step-title-button"
                                       aria-expanded="true" aria-controls="step-panel-2">
                                   <span class="js-step-title-text">
-                                    Pupils that have not started the check yet
+                                    Pupils that have not completed the check
```

Comment:
Does the text and the code agree here?  This is starting to feel like a spec change.